src/Integration.UnitTests/Suppression/SonarQubeIssuesProviderTests.cs

```diff
@@ -467,44 +401,49 @@ private void ExecuteGetIssuesOnTriggerWithError(string projectId, Action trigger
                 fetchCallCount++;
                 throw new ApplicationException("dummy error from mock");
             };
+            SetupSolutionBinding(isConnected: true, serviceFetchIssuesTask: serviceFetchIssuesTask);
 
-            mockSqService.Setup(x => x.GetSuppressedIssuesAsync(It.IsAny<string>(), It.IsAny<CancellationToken>()))
-                .ReturnsAsync(serviceFetchIssuesTask)
-                .Verifiable();
-
-            triggerFetchIssues();
+            RaiseTimerElapsed(DateTime.UtcNow);
 
             // 3. Fetch issues again - should used cached issues
-            matches = issuesProvider.GetSuppressedIssues(projectId, "folder1/file1");
+            matches = issuesProvider.GetSuppressedIssues("projectId", "folder1/file1");
             VerifyServiceGetIssues(Times.Exactly(2));
             matches.Count().Should().Be(1);
             fetchCallCount.Should().Be(1);
+
+            VerifyTimerStart(Times.Exactly(1)); // once, on construction
         }
 
         #endregion
 
         /// <summary>
-        /// Configures the mock tracker and service to return the specified values
+        /// Configures the mock service to return the specified values
         /// </summary>
-        private void SetupSolutionBinding(bool isBound, bool isConnected, string projectKey, IList<SonarQubeIssue> issues)
```

Comment:
We no longer need the project key: we just configure the mock to return the specified issues.
There is a separate assertion method that the tests can use to check the correct SQ key way supplied.